source/_docs/guides/drupal-8-commerce.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Drupal Commerce on Drupal 8
+description: Use Composer to Install Drupal Commerce on Pantheon, using Drupal 8
+contributors: [alexfornuto, stevector]
+tags: [drupal, siteintegrations, moreguides]
+type: guide
+permalink: docs/guides/:basename/
+---
+
+This guide covers installing Drupal 8 Commerce, a Drupal implementation designed specifically for e-commerce. At the end of this guide you will have a Drupal Commerce site, GitHub repository, and Circle CI configuration for testing.
+
+
+## Before You Begin
+
+This process uses Composer to manage modules and dependencies. Before proceeding, you may wish to consult the following docs:
+
+ - [Composer Fundamentals and Workflows](/docs/composer)
+ - [Build Tools](/docs/guides/build-tools)
+
+
+In addition to Pantheon, you will need accounts at:
+
+ - [GitHub](https://github.com){.external}
+ - [CircleCI](https://circleci.com){.external}
+
+
+1.  Follow the [Before You Begin](/docs/guides/build-tools/#before-you-begin) section of the Build Tools guide to install Composer, Terminus, and the Terminus Build Tools plugin on your local computer, and create machine tokens for [GitHub](https://help.github.com/articles/creating-an-access-token-for-command-line-use/){.external} and [CircleCI](https://circleci.com/account/api){.external}. Export the tokens to your current terminal session, as described below.
+
+2.  This guide uses several variables in example [Terminus](/docs/terminus) commands. This lets you copy and paste without needing to change the variable. For this to work, you must first export the variables in your local terminal session:
+
+    ```bash
+    export SITENAME=yoursitenamehere
+    export GITHUB_TOKEN=yourgithubtokenhere
+    export CIRCLE_TOKEN=yourcirclecitokenhere
+    ```
+
+## Create A New Drupal 8 Site
+
+1.  Using the Terminus Build Tools plugin, create a new Drupal 8 site from the Pantheon [Example Drops 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer) repository on GitHub:
+
+    ```bash
+    terminus build:project:create pantheon-systems/example-drops-8-composer $SITENAME
+    ```
+    At this point *do not* go to the web interface to continue installation.
+
+2.  Use Terminus to retrieve the Git URL for your new site and pass it to an environment variable:
```

Comment:
I think this step can be removed. `GITURL` is not used later in the guide.

Reply:
`GITURL` is used in the next step to clone the project, on L56

Reply:
The clone is done from GitHub. This url is Pantheon.

Reply:
Excuse me, I'll be right back...

👊 🤕 

Yup...